static/bonecard.css

```diff
@@ -146,8 +147,8 @@ div.bonecard-micro:before {
 }
 
 .bonecard-code .code {
-    width: 834px;
-    height: 527px;
+    width: 844px;
```

Comment:
Why the new dimensions? Goal was for this to be "phone friendly". Please use resolutions that will be good for that.

Reply:
I changed the dimensions to try to fix this issue : https://github.com/jadonk/bone101/issues/28 , will revert the change